resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -24,6 +24,10 @@
             {
                 "syscall": "write"
             },
+            {
+                "syscall": "writev",
+                "comment": "Used by the VirtIO net device to write to tap"
```

Comment:
In the MMDS path, we are still doing an extra copy. I evaluated the possibility to modify the MMDS Network stack to use `IoVecBuffer`s but I think this would complicate the code quite a lot and given that MMDS is not on the critical path, I think it's not worth the effort. What do you think?

Reply:
Agreed, thanks for the details!